config/envmapprovider/mapprovider_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package configmapprovider
+package envmapprovider
```

Comment:
Is this the same as the file map provider? Looks pretty much the same.

Reply:
Ha? no, this reads from env vars, the other reads files. Also the file provider should soon implement file watch.